code/fmriprep/fmriprep.sbatch

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+# Copyright 2024 The Axon Lab <theaxonlab@gmail.com> 
+# 
+# Licensed under the Apache License, Version 2.0 (the "License"); 
+# you may not use this file except in compliance with the License. 
+# You may obtain a copy of the License at 
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0 
+# 
+# Unless required by applicable law or agreed to in writing, software 
+# distributed under the License is distributed on an "AS IS" BASIS, 
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+# See the License for the specific language governing permissions and 
+# limitations under the License. 
+###########################################################################
+#
+# General SLURM settings
+#SBATCH --mail-type=ALL 
+#SBATCH --mail-user=<email>@unil.ch
```

Comment:
Let's create a secret setting:
```suggestion
#SBATCH --mail-user={{ secrets.data.curnagl_email | default('<email>@unil.ch') }}
```

Reply:
This does not work as we don't share the same secret (our email address are different).
Let's leave it as such. Also, that's how you put it in fmriprep_anatonly.sbatch